src/main/java/edu/umich/tl/ApiCallHandler.java

```diff
@@ -101,7 +173,10 @@ private HttpResponse apiDirectCanvas(String url) {
 				clientRequest.addHeader(h.getName(), h.getValue());
 			}
 			try {
+				Stopwatch stopwatch = Stopwatch.createStarted();
 				httpResponse = client.execute(clientRequest);
+				stopwatch.stop();
+				M_log.info("The Api call \"" + url+ "\" took \""+stopwatch+"\" and ResponseCode: "+httpResponse.getStatusLine().getStatusCode());
```

Comment:
It's nice to have units for the stopwatch output.  Perhaps that is put in automatically?